scopes/workspace/variants/variants.docs.md

```diff
@@ -0,0 +1,127 @@
+---
+id: variants
+title: Variants
+slug: /aspects/variants
+---
+
+The `teambit.workspace/variants` extension enables a cascading, CSS-like, selection of components in the workspace configuration. It provides order and simplicity to the way we apply rules and settings to large number of independent components - all in a single workspace.
+
+Configurations set on a specific set of components will:
```

Comment:
maybe it's also worth mentioning the relation between compoennts.json and variants